src/chrome/content/rules/Frugalware.org.xml

```diff
@@ -0,0 +1,38 @@
+<!--
+	Invalid certficate:
+		darcs.frugalware.org
+		dors.frugalware.org
+		git.frugalware.org
+		lists.frugalware.org
+		packages.frugalware.org
+		planet.frugalware.org
+		wiki.frugalware.org
+
+	Private subdomain:
+		webmail.frugalware.org
+
+-->
+<ruleset name="Frugalware.org">
+
+	<target host="frugalware.org" />
+	<target host="www.frugalware.org" />
+	<target host="bugs.frugalware.org" />
+	<target host="lists.frugalware.org" />
+	<target host="packages.frugalware.org" />
+	<target host="planet.frugalware.org" />
+
+	<securecookie host="^.+\.frugalware\.org$" name=".+" />
+
+	<rule from="^http://lists\.frugalware\.org/"
+		to="https://frugalware.org/mailman/listinfo/" />
+
+	<rule from="^http://packages\.frugalware\.org/"
+		to="https://frugalware.org/packages/" />
+
+	<rule from="^http://planet\.frugalware\.org/"
+		to="https://frugalware.org/" />
```

Comment:
e.g. http://planet.frugalware.org/index.php?post_id=27 will not work with this rule.